test/parallel/test-os.js

```diff
@@ -116,7 +116,8 @@ const interfaces = os.networkInterfaces();
 switch (platform) {
   case 'linux':
   {
-    const filter = (e) => e.address === '127.0.0.1';
+    const filter =
+      (e) => e.address === '127.0.0.1' && e.netmask === '255.0.0.0';
     const actual = interfaces.lo.filter(filter);
     const expected = [{ address: '127.0.0.1', netmask: '255.0.0.0',
```

Comment:
[warning 🚨 scope-creep]
AFAICT `expected` and the asserting could be hoisted, and `switch` replaced with `if` or trinary for selecting `interfaces.lo` / `interfaces['Loopback Pseudo-Interface 1']`